src/sdk/model/common/__json__/entity-type.ts

```diff
@@ -1,18 +1,72 @@
 /**
  * Enumerates the core entity types.
  */
-export type EntityType = 'COMPANY' |
-  'ORG' |
-  'VDC' |
-  'VAPP' |
-  'VM' |
-  'CATALOG' |
-  'VPG' |
-  'ORG_VDC_NETWORK' |
-  'EDGE' |
-  'VAPP_NETWORK' |
-  'VAPP_TEMPLATE' |
-  'MEDIA' |
-  'EXTERNAL_NETWORK' |
-  'VCC_TENANT' |
-  'COMPANY';
+export type EntityType =
```

Comment:
We would need to change those type to enum at some points. Just comment on this to no forget about it.